app/src/main/res/layout/activity_main.xml

```diff
@@ -7,7 +7,7 @@
     tools:context=".MainActivity">
 
     <TextView
-        android:id="@+id/textView"
+        android:id="@+id/txt_game_name"
```

Comment:
Why ? I do not undestand this change in the Main activity

Reply:
I don't either, and I don't remember doing this change, will revert